projectName-web/src/test/java/xxxxxx/yyyyyy/zzzzzz/app/welcome/WelcomeControllerTest.java

```diff
@@ -0,0 +1,35 @@
+package xxxxxx.yyyyyy.zzzzzz.app.welcome;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.Locale;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.validation.support.BindingAwareModelMap;
+
+/**
+ * Executes the test of handling requests for the application home page.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath:META-INF/spring/testContext.xml" })
+public class WelcomeControllerTest {
```

Comment:
No need to add unit test for the controller.
Add only a testcase for the "integration test".